ml_tools/previewer.py

```diff
@@ -216,16 +218,23 @@ def add_regions(self, draw, regions, v_offset=0):
             draw.rectangle(self.rect_points(rect, v_offset), outline=(128, 128, 128))
 
     def add_tracks(
-        self, draw, tracks, frame_number, track_predictions=None, screen_bounds=None
+        self,
+        draw,
+        tracks,
+        frame_number,
+        track_predictions=None,
+        screen_bounds=None,
+        colours=None,
```

Comment:
could this just be
```colours=TRACK_COLOURS,```

and remove the if not colours: ....

Reply:
Of course.   Thanks for the tip.